src/SFA.DAS.LevyTransferMatching.Web/Views/Pledges/Close.cshtml

```diff
@@ -0,0 +1,75 @@
+                <div class="govuk-form-group" das-highlight-error-for="HasConfirmed" error-class="govuk-form-group--error" id="HasConfirmed">
+                    <fieldset class="govuk-fieldset">
+
+                        <legend class="govuk-fieldset__legend">
+                            <h1 class="govuk-fieldset__heading govuk-label--s">Would you like to continue and close this pledge?</h1>
+                        </legend>
+
+                        <span das-validation-for="HasConfirmed" class="govuk-error-message"></span>
+
+                        <div class="govuk-radios govuk-radios--conditional" data-module="govuk-radios">
+
+                            <div class="govuk-radios__item">
+                                <input class="govuk-radios__input" id="close-pledge-yes" asp-for="HasConfirmed" value="true" type="radio" data-aria-controls="conditional-close-pledge-yes" das-highlight-error-for="HasConfirmed" error-class="govuk-textarea--error">
+                                <label class="govuk-label govuk-radios__label" for="close-pledge-yes">Yes, close pledge @Model.EncodedPledgeId</label>
+                            </div>
+
+                            <div class="govuk-radios__conditional govuk-radios__conditional--hidden" id="conditional-close-pledge-yes">
+
+                                <div class="govuk-form-group">
+
+                                    <div class="govuk-form-group">
+
+                                        <label class="govuk-label">
+                                            <p><strong>Disclaimer: </strong></p>
+                                            <p>Your pledge will no longer be shown publicly, and businesses won't be able to apply for funding from it.</p>
+                                            <p>This pledge will continue to fund all applications you have already approved, including applications you have approved but the applicant has not yet accepted funding.</p>
+                                            <p>Though closed, you will still be able to view, approve and reject any applications you have received for this pledge.</p>
+                                        </label>
+
+                                    </div>
+
+                                </div>
+
+                            </div>
+                            <div class="govuk-radios__item">
+                                <input class="govuk-radios__input" id="close-pledge-no" asp-for="HasConfirmed" value="false" type="radio" das-highlight-error-for="HasConfirmed" error-class="govuk-textarea--error">
+                                <label class="govuk-label govuk-radios__label" for="close-pledge-no">No, keep pledge @Model.EncodedPledgeId active</label>
+                            </div>
+                        </div>
+                    </fieldset>
+                </div>
+
+                <div class="govuk-button-group">
+
+                    @if (Model.UserCanClosePledge)
```

Comment:
Can the user get to this page if they can't close the pledge? It seems odd to hide the option

Reply:
Need some advice from you, Chris.
According to AC1 of this story, 
**AC1: Allow an account 'Owner' or 'Transactor' to close a pledge**
Given that a sending employer account user is an 'Owner' or 'Transactor'
When they are viewing the application list page
Then display a 'Close pledge' option

so, this property is basically used to hide and show the Close Pledge button. 
1) Close Pledge button will not be visible when "Pledge Status" is already closed
2) Close Pledge button will not be visible when the user is not Owner Or Transactor.

UserCanClosePledge = result.PledgeStatus != PledgeStatus.Closed && _userService.IsOwnerOrTransactor(request.AccountId)

Reply:
That looks correct to me :+1: